src/components/templates/Camp/components/Map.tsx

```diff
@@ -29,7 +29,7 @@ import "./Map.scss";
 
 interface MapProps {
   venue: CampVenue;
-  partygoers: readonly WithId<User>[];
+  partygoers: WithId<User>[];
```

Comment:
For my knowledge, why did we remove the `readonly` from this one?

Reply:
Was changed back